Source/App.js

```diff
@@ -1 +1,417 @@
+            var name = entity.id;
+
+            if (Cesium.defined(entity.polygon)) {
+                //Set the polygon material to a random, translucent color.
+                entity.polygon.material = Cesium.Color.fromRandom({
+                    red : 0.1,
+                    maximumGreen : 0.5,
+                    minimumBlue : 0.5,
+                    alpha : 0.6
+                });
+                // Make the polygon conform to terrain.
+                entity.polygon.height = undefined;
+                entity.polygon.heightReference = Cesium.HeightReference.CLAMP_TO_GROUND;
+                // Add to the neighborhoods group
+                entity.parent = neighborhoods;
+                // Generate Polygon center
+                var poly_center = Cesium.BoundingSphere.fromPoints(entity.polygon.hierarchy.getValue().positions).center;
+                poly_center = Cesium.Ellipsoid.WGS84.scaleToGeodeticSurface(poly_center);
+                entity.position = poly_center;
+                // Generate labels
+                entity.label = new Cesium.LabelGraphics({
+                    text : entity.kml.extendedData.ntaname.value,
+                    showBackground : true,
+                    scale : 0.6,
+                    horizontalOrigin : Cesium.HorizontalOrigin.CENTER,
+                    verticalOrigin : Cesium.VerticalOrigin.BOTTOM,
+                    distanceDisplayCondition : distanceDisplayCondition,
+                    disableDepthTestDistance : Number.POSITIVE_INFINITY
+                });
+            }
+        }
+        neighborhoods.show = false;
+    });
+
+    var points = viewer.entities.add(new Cesium.Entity());
+    pointsPromise.then(function(dataSource) {
+        // Add the new data as entities to the viewer
+        viewer.dataSources.add(dataSource);
+
+        // Get the array of entities
+        var entities = dataSource.entities.values;
+
+        for (var i = 0; i < entities.length; i++) {
+            var entity = entities[i];
+            if (Cesium.defined(entity.billboard)) {
+                // Add to the points group
+                entity.parent = points;
+
+                // Add distance display condition
+                entity.billboard.distanceDisplayCondition = distanceDisplayCondition;
+
+                // modify description
+                var cartographicPosition = Cesium.Cartographic.fromCartesian(entity.position.getValue(Cesium.JulianDate.now()));
+                var latitude = Cesium.Math.toDegrees(cartographicPosition.latitude);
+                var longitude = Cesium.Math.toDegrees(cartographicPosition.longitude);
+
+                var description = '<table class="cesium-infoBox-defaultTable cesium-infoBox-defaultTable-lighter"><tbody>';
+                description += '<tr><th>' + "Latitude" + '</th><td>' + latitude + '</td></tr>';
+                description += '<tr><th>' + "Longitude" + '</th><td>' + longitude + '</td></tr>';
```

Comment:
Do we have more interesting information to show here instead of just latitude and longitude?